dist/stress.d.ts

```diff
@@ -48,10 +48,10 @@ export declare class Stress {
     /**
      * Constructor allows for construction with a bunch of optional parameters
      *
-     * @param runtime - see {@link StressOptionsType}.
-     * @param dop - see {@link StressOptionsType}.
-     * @param iterations - see {@link StressOptionsType}.
-     * @param passThreshold - see {@link StressOptionsType}.
+     * @param runtime - see {@link StressOptions}.

```

Comment:
This was just a typo correction as the corresponding interface was called StressOptions